Azure/sonic-buildimage
# [Fast-Reboot]: FR mode is active only first 3 minutes after start.

**- What I did**
Limit Fast-Reboot mode by 3 minutes after reboot.

**- How I did it**
When syncd and quagga are started we set Fast-Reboot mode start iff uptime is less than 3 minutes.

**- How to verify it**
After fast-reboot wait 5 minutes and restart syncd. You should see that syncd starts in cold boot mode.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
